<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>361</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="fileProperties">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpen">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCompress">
       <property name="text">
        <string>Compress</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="icon">
     <iconset theme="go-previous">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btnFront">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>10</y>
      <width>41</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
    <property name="icon">
     <iconset theme="go-next">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentPath">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset theme="search">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHome">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
    <property name="icon">
     <iconset theme="home">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackCheck">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Background Check</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionThresholdSize"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionFunctions"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFunctions">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionThresholdSize">
   <property name="text">
    <string>Threshold Size</string>
   </property>
  </action>
  <action name="actionCheck_Time">
   <property name="text">
    <string>Check Time</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
